developing-with-nats/tutorials/jwt.md

```diff
@@ -0,0 +1,1540 @@
+
+##### Activations
+
+To revoke all activations of the export, identified by `--account` and `--subject` (`--stream` it the export is a stream), issued for a given Account identity NKEY use: `nsc revocations add-activation --account <account name> --subject <export name> --target-account <account identity public NKEY>`
+Use `nsc revocations list-activations --account SYS` to inspect the result or `nsc revocations delete_activation --account <account name> --subject <export name> --target-account <account identity public NKEY>` to remove the revocation.
+
+```text
+> nsc revocations add-activation --account SYS --subject foo --target-account AAUDEW26FB4TOJAQN3DYMDLCVXZMNIJWP2EMOAM5HGKLF6RGMO2PV7WP
+[ OK ] revoked activation "foo" for account AAUDEW26FB4TOJAQN3DYMDLCVXZMNIJWP2EMOAM5HGKLF6RGMO2PV7WP
+> nsc revocations list-activations --account SYS
++------------------------------------------------------------------------------------------+
+|                             Revoked Accounts for stream foo                              |
++----------------------------------------------------------+-------------------------------+
+| Public Key                                               | Revoke Credentials Before     |
++----------------------------------------------------------+-------------------------------+
+| AAUDEW26FB4TOJAQN3DYMDLCVXZMNIJWP2EMOAM5HGKLF6RGMO2PV7WP | Wed, 10 Feb 2021 13:22:11 EST |
++----------------------------------------------------------+-------------------------------+
+```
+
+Please note that the revocation created only applies to JWT issued before the time listed. 
+Activations created or edited after, will be valid as they are outside of the revocation time.
+Please be also aware that adding a revocation will modify the account and therefore has to be pushed in order to publicize the revocation.
+
+##### Accounts
+
+Account identity NKEYS can not be revoked like user or activations. 
+Instead lock out all users by setting the connection count to 0 using `nsc edit account --name <account name> --conns 0` and pushing the change using `nsc push --all`.
+
+Alternatively you can also remove the account using `nsc delete account --name` and keep it from found by the account resolver. How to do this depends on your resolver type:
+* [mem-resolver](https://docs.nats.io/nats-server/configuration/securing_nats/jwt/resolver#memory): Remove the JWT from the configuration field `resolver_preload` and restart all `nats-server`
+* [url-resolver](https://docs.nats.io/nats-server/configuration/securing_nats/jwt/resolver#url-resolver): Manually delete the JWT from the `nats-account-server` store directory.
+* `nats-resolver`: Prune removed accounts using: `nsc push --all --prune`. For this to work, the resolver has to have deletion enabled (`allow_delete: true`) and you need to be in possession of an operator signing key.
+
+##### Signing keys
```

Comment:
soon we need to add a section on the `scoped` signing keys

Reply:
once we add them to nsc I'll add them here